src/Engine/babylon.engine.ts

```diff
@@ -613,7 +613,7 @@
         /**
          * Observable event triggered each time the canvas receives pointerout event
          */
-        public onCanvasPointerOutObservable = new Observable<Engine>();
+        public onCanvasPointerOutObservable = new Observable<PointerEvent>();

```

Comment:
I'm fine with the change. Please just add it in the breaking changes list in the what'snew.md

Reply:
added